clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -145,16 +149,29 @@ public void start() throws Exception {
         logger.log(LogLevel.Information, "HubConnected started");
     }
 
+
     /**
      * Stops a connection to the server.
      */
-    public void stop(){
-        logger.log(LogLevel.Debug, "Stopping HubConnection");
+    public void stop(String errorMessage) {
```

Comment:
The other clients don't allow the user to stop with an error message, so I don't think we should. The version with an error message should be `private`.